package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "barrels",
-  "version": "1.6.5",
+  "version": "1.6.7",
```

Comment:
Why are you skipping a version? 🤔

Reply:
made changes twice, so upped the version twice. downgraded the version on my code now, so it will be upped only once on your side.